.pylintrc

```diff
@@ -4,4 +4,4 @@ disable=duplicate-code,too-many-public-methods,too-few-public-methods,fixme
 
 [BASIC]
 
-good-names=i, j, k, v, e, g, id, _, __
+good-names=i, j, k, v, e, g, id, _, __, t, r
```

Comment:
I would remove `t` and `r` from this list. All the other variable names are very common and usually have well-defined meanings. That's not the case for `t` and `r`. For instance,

- `i`, `j`: iteration indices
- `k`, `v`: key value when iterating maps
- `g`: graph
- `id`: id
- `_`: discarded value
- `__`: anonymous traversal

If you want to use `t` and `r` somewhere because you copied code from Altimeter, then disable this control only for the specific class.